addons/uis/apps/android.md

```diff
@@ -32,6 +32,11 @@ The latest release version of the app is always available through Google Play.
   <div class="col s12 m5"><img src="images/android_02.png" alt="Demo Widget Overview"></div>
 </div>
 
+**Getting Started:**
+
+When first installed the app is in demo mode. To connect the android app to your openhab server, first navigate to Settings and uncheck the Demo mode option.
```

Comment:
This is indeed nasty that it is required (but I agree that atm it should be in the docs) - my hope is that https://github.com/openhab/openhab.android/issues/179 gets implemented one day.